### PR TITLE
load info for endpoints

### PR DESCRIPTION
**Pull request by PashaKlybik**
staking/eraElectionStatus
staking/forceEra

### CODE REVIEW

File: src/substrate_query_interface.js [hunk 1]
```diff
@@ -101,6 +101,24 @@ async function getStakingActiveEra(api) {
         Timeout.set(TIMEOUT_TIME_MS, 'API call staking/activeEra failed.')]);
 }
 
+async function getValidatorCount(api) {
```
Comment (dillu24) [inline src/substrate_query_interface.js:104]: Should be renamed to `getStakingValidatorCount`

File: src/substrate_query_interface.js [hunk 2]
```diff
@@ -101,6 +101,24 @@ async function getStakingActiveEra(api) {
...
+    return await Promise.race([
+        api.query.staking.validatorCount(),
+        Timeout.set(TIMEOUT_TIME_MS, 'API call staking/validatorCount failed.')]);
+}
+
+async function getEraElectionStatus(api) {
```
Comment (dillu24) [inline src/substrate_query_interface.js:110]: should be renamed to `getStakingEraElectionStatus`

File: src/substrate_query_interface.js [hunk 3]
```diff
@@ -101,6 +101,24 @@ async function getStakingActiveEra(api) {
...
+    return await Promise.race([
+        api.query.staking.eraElectionStatus(),
+        Timeout.set(TIMEOUT_TIME_MS, 'API call staking/eraElectionStatus failed.')]);
+}
+
+async function getForceEra(api) {
```
Comment (dillu24) [inline src/substrate_query_interface.js:116]: should be renamed to `getStakingForceEra`

**Comment by dillu24** [comment]
First of all we would like to thank you for your contribution. We are very eager to add your implementation to our code base.

Apart from the changes I suggested in my review, I propose 2 other changes:

1. Add some calls to your functions in the `demos/api_demo.py` file (follow the examples in that file)
2. We will wait untill we merge branch `dylan/v1.19.1`. The reason is that, that branch contains a lot of refactoring. So i suggest we either wait for that branch to be merged, and modify your implementation as per the implementation in that branch, or you modify your implementation to follow the implementation in branch `dylan/v1.19.1`, and merge into branch `dylan/v1.19.1`.

File: src/server.js [hunk 4]
```diff
@@ -697,6 +697,78 @@ async function startPolkadotAPI() {
         }
     });
 
+    app.get('/api/query/staking/validatorCount', async function (req, res) {
```
Comment (DanielMagro97) [inline src/server.js:700]: Kindly reorder these as follows:
1. activeEra
2. eraElectionStatus
3. erasRewardPoints
4. erasStakers
5. erasTotalStake
6. erasValidatorReward
7. forceEra
8. validatorCount

To keep order in line with that of [the docs](https://polkadot.js.org/api/substrate/storage.html#staking)

The same order should also be kept in `substrate_query_interface.js`

**Comment by dillu24** [comment]
@PashaKlybik branch `dylan/v1.19.1` has been merged to master. I suggest that you take a look at the changes to perform the necessary fixes for this PR to be merged.

**Comment by dillu24** [comment]
Given that there are a number of conflicts and a lot of time passed, I am going to close this PR.

**Pull request closed without merging**